### ISSUE TITLE
[Accessibility][List][Sample CSS] Links are only visibly discernible from surrounding text because of color 

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1681]
**Steps to Reproduce**
1. Open the page in Chrome browser.
2. View list with links and use a contrast tool to identify contrast ratio between link text and body text.

**Current Behavior**
Links in the list with linked items are only visually discernible from the following text because of their color.

The contrast ratio between link text (#1473E6) and surrounding text (#737373) is 1:1. Users with low vision or who have difficultly perceiving color may be unable to distinguish link text from the surrounding text.

**Expected behavior/code**
Links should be visibly discernible by means other than color. The clearest visual distinction for links is an underline. An alternative visual distinction is the use of a color that has a significant difference in lightness/relative luminance from the surrounding text (i.e. a difference with a contrast ratio of 3:1 or greater). Both link text and surrounding text must still meet a contrast ratio of 4.5:1 against the background.

**Environment**
Chrome 91, Windows 10
AEM Core Components v. 2.17  

**Affected Users**
Users with low vision, users without perception of color

**WCAG SC**
1.4.1 Use of Color

**Comment by bpauli** [comment]
@adobe export issue to Jira project SITES

**Comment by gabrielwalt** [comment]
Let's add an underline to all blue link styles in the Component Library.
